This file is part of mingw-cross-env.
See doc/index.html for further information.

This patch has been proposed to upstream:
http://www.gnu.org/software/gsasl/#bugs

--- a/libgsasl.pc.in
+++ b/libgsasl.pc.in
@@ -15,4 +15,5 @@
 URL: http://www.gnu.org/software/gsasl/
 Version: @VERSION@
 Libs: -L${libdir} -lgsasl
+Libs.private: @LTLIBGCRYPT@ @LTLIBIDN@ @LTLIBNTLM@
 Cflags: -I${includedir}
